test/metabase/db/load_and_dump_test.clj

```diff
@@ -0,0 +1,51 @@
+(ns metabase.db.load-and-dump-test
+  (:require [clojure.test :refer :all]
+
+            [metabase.cmd.migrate :as migrate]
+            [metabase.cmd.load-from-h2 :as load-from-h2]
+            [metabase.cmd.dump-to-h2 :as dump-to-h2]
+            [metabase.driver.sql-jdbc.connection :as sql-jdbc.conn]
+            [metabase.test.data.interface :as tx]
+            [metabase.db.connection :as mdb.connection]
+            [metabase.test :as mt]
+            [metabase.driver :as driver]
+            [metabase.cmd.compare-h2-dbs :as compare-h2-dbs]
+            [metabase.cmd.copy.h2 :as h2]
+
+            [metabase.models.user :refer [User]]
+            [metabase.test.util :as tu]
+            [toucan.db :as db]))
+
+(defn- abs-path
+  [path]
+  (.getAbsolutePath (clojure.java.io/file path)))
+
+(deftest load-and-dump-test
+  (testing "loading data to h2 and porting to DB and migrating back to H2"
+    ;; do we import clojure.java.io even for just one call?
+    (let [h2-fixture-db-file (abs-path "frontend/test/__runner__/test_db_fixture.db")
+          h2-file (abs-path "frontend/test/__runner__/test_db_fixture2.db")
+          db-name "dump-test"]
+
+      ;; XXX how the :mysql works if I don't have it running?
+      (mt/test-drivers #{:postgres :mysql}
+
```

Comment:
https://guide.clojure.style/#no-blank-lines-within-def-forms

I think there are a few exceptions:
- breaking out a giant `deftest` form into a few different `testing` sections
- breaking out different arities of a `defn` form (this is used as a "bad" example but I don't think it's that bad)

either way, there shouldn't be any blank lines here